examples/basic/print.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+
+	"gopkg.in/src-d/go-cli.v0"
+)
+
+func init() {
+	app.AddCommand(&printCommand{})
+}
+
+type printCommand struct {
+	cli.Command
+	_ string `name:"print" short-description:"prints a message" long-description:"prints a very nice message"`
```

Comment:
This feels a bit weird. What do you think about this struct tag being in the `cli.Command` embedded field?

Something like:

```go
type printCommand struct {
        cli.Command `name:"print" short-description:"prints a message" long-description:"prints a very nice message"`
        // ...
}
```

Reply:
Oh, that indeed would look better. It needs another struct (e.g. `cli.PlainCommand`), since `cli.Command` provides some defaults such as logging and profiling that are not used for commands like `version`.

Reply:
Done. It is now mandatory to embed `PlainCommand`. This also works if it is embedded indirectly through `Command` or even your own custom `baseCommand`.